assets/prototype/application/utilities/text/entityFieldSearch/index.ts

```diff
@@ -27,11 +27,11 @@ type EntityFieldSearchProps = {
 const entityFieldSearch = ({ entities, searchText }: EntityFieldSearchProps): Entity[] => {
 	const isSearchable = searchText && trim(searchText).length && Array.isArray(entities) && entities.length;
 	const textToSearch = trim(searchText).toLowerCase();
-	const searchResult =
-		isSearchable &&
-		entities.filter(({ name: title }) => {
-			return title && title.toLowerCase().search(textToSearch) !== -1;
-		});
+	const searchResult = isSearchable
+		? entities.filter(({ name: title }) => {
+				return title && title.toLowerCase().search(textToSearch) !== -1;
+		  })
+		: [];
```

Comment:
Fix for:
![image](https://user-images.githubusercontent.com/1477453/72363816-25ae6280-36fe-11ea-9457-e99aae6baf21.png)